product-catalog/test/productcatalog/services/CategoryServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package productcatalog.services;
+
+import common.categories.CategoryUtils;
+import common.products.ProductUtils;
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.categories.CategoryTree;
+import io.sphere.sdk.products.ProductProjection;
+import org.junit.Test;
+import productcatalog.exceptions.ProductNotFoundException;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CategoryServiceTest {
+
+    private final CategoryTree categories = CategoryTree.of(CategoryUtils.getQueryResult("categoryQueryResult.json").getResults());
+    private final List<ProductProjection> products = ProductUtils.getQueryResult("productProjectionQueryResult.json").getResults();
+    private final CategoryService categrieService = new CategoryService(categories);
+
+    @Test
+    public void getSiblingCategories() throws ProductNotFoundException {
+        final ProductProjection suttonBag = products.stream()
+                .filter(projection -> projection.getId().equals("254f1c0e-67bc-4aa6-992d-9a0fea1846b5"))
+                .findFirst().get();
+
+        final Category clutches = categories.findById("a9c9ebd8-e6ff-41a6-be8e-baa07888c9bd").get();
+        final Category satchels = categories.findById("30d79426-a17a-4e63-867e-ec31a1a33416").get();
+        final Category shoppers = categories.findById("bd83e288-77de-4c3a-a26c-8384af715bbb").get();
+        final Category handBags = categories.findById("9a584ee8-a45a-44e8-b9ec-e11439084687").get();
+        final Category wallets = categories.findById("d2f9a2da-db3e-4ee8-8192-134ebbc7fe4a").get();
+        final Category backpacks = categories.findById("46249239-8f0f-48a9-b0a0-d29b37fc617f").get();
+        final Category shoulderBags = categories.findById("8e052705-7810-4528-ba77-00094b87a69a").get();
```

Comment:
Same for all the categories. Actually, I would write the test showing:
- AssertThat product has X category
- AssertThat X category has Y,Z,W siblings
- AssertThat the method returns Y,Z,W categories
To make it evident the criteria to choose the list of categories.